fe/src/main/java/org/apache/doris/load/loadv2/LoadJobScheduler.java

```diff
@@ -47,6 +51,10 @@
 
     private LinkedBlockingQueue<LoadJob> needScheduleJobs = Queues.newLinkedBlockingQueue();
 
+    // Used to implement a job granularity lock for update global dict table serially
+    // The runningBitmapTables keeps the doris table id of the spark load job which in state pending or etl
+    private Map<Long, Set<Long>> runningBitmapTableMap = new HashMap<>();
```

Comment:
Why calling it a "bitmap" table?

Reply:
Maybe "runningTableWithBitmapColumns" is better ?